CSharp/BatchExplorer/Models/ComputeNodeModel.cs

```diff
@@ -258,7 +306,7 @@ public override List<PropertyModel> PropertyModel
         {
             try
             {
-                if (this.ComputeNode.SchedulingState != SchedulingState.Disabled)
+                if (this.ComputeNode.SchedulingState != Microsoft.Azure.Batch.Common.SchedulingState.Disabled)
```

Comment:
Yes because there is a property of the same name that the compiler attempts to resolves to first.